### ISSUE TITLE
pyplotlib

### ISSUE THREAD

**Issue by Jimmy586** [Jimmy586/MachineLearningJourney#1]
![image](https://user-images.githubusercontent.com/31384852/155826474-af26dec1-19db-48e5-9c45-92d2a400fdfe.png)

**Comment by Jimmy586** [comment]
the blue line is obtained after 1500 iteration of running the gradient descent function with 0.2 of learning rates.
